src/_includes/backward-incompatible-changes/open-source/2.3.0-2.4-develop.html

```diff
@@ -1041,6 +1081,30 @@ <h3 id="2.3.0-2.4-develop-database">Database changes</h3>
         <th>What changed</th>
         <th>How it changed</th>
     </tr>
+    <tr>
+        <td>signifyd_case</td>
```

Comment:
More duplicates here to 1107?

Reply:
see comments to `commerce/2.3.0-2.4-develop.html` on line 849